Services/Movies/Movies.API/Controllers/MoviesController.cs

```diff
@@ -40,7 +40,27 @@ public async Task<IActionResult> GetMoviesByPerson(int id)
             var movies = await _repository.GetMoviesByPerson(id);
             return Ok(_mapper.Map<List<Movie>, List<MovieModel>>(movies));
         }
+        
+        [HttpGet("[action]")]
+        public async Task<IActionResult> FilterMovies([FromQuery] FilterMoviesRequest request)
+        {
+            var (movies, count) = await _repository.FilterMovies(request);
+            return Ok(new PaginatedMovieList {Movies = _mapper.Map<List<Movie>, List<MovieModel>>(movies), Count = count});
```

Comment:
Mapper moze sam da zakljuci tip iz koga konvertujes. Dovoljno je napisati `_mapper.Map<List<MovieModel>>(movies)`. Ako hoces promeni to, cini mi se citljivije